osu.Game/Screens/Play/Player.cs

```diff
@@ -460,7 +460,7 @@ protected virtual ScoreInfo CreateScore()
         {
             var score = new ScoreInfo
             {
-                Beatmap = Beatmap.Value.BeatmapInfo,
+                Beatmap = gameplayBeatmap.BeatmapInfo,
```

Comment:
Causes crashes [as discussed on Discord](https://discord.com/channels/188630481301012481/188630652340404224/723926724244537424) as it looks the gameplay beatmap's instance is not necessarily attached.